### PR TITLE
SAK-46272 calendar wc make calendar widget display all events

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-46272

### CODE REVIEW

File: calendar/calendar-impl/impl/pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>sakai-calendar-hbm</artifactId>
             <version>${project.version}</version>
-        </dependency>
+		</dependency>
```
Comment (jesusmmp) [inline calendar/calendar-impl/impl/pom.xml:61]: Undo change

File: calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java [hunk 2]
```diff
@@ -458,6 +426,10 @@ public List<CalendarEvent> getFilteredEvents(Map<EventFilterKey, Object> options
 		if (options.containsKey(EventFilterKey.SITE)) {
 			// A single site has been requested
 			allRefs.add(calendarReference((String) options.get(EventFilterKey.SITE), SiteService.MAIN_CONTAINER));
+        } else if (options.containsKey(EventFilterKey.SITES)) {
```
Comment (jesusmmp) [inline calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java:429]: Spacing

File: calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventDetails.java [hunk 3]
```diff
@@ -19,8 +19,8 @@
 import java.util.List;
 
 import lombok.Data;
-
 import lombok.EqualsAndHashCode;
+
```
Comment (jesusmmp) [inline calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventDetails.java:23]: Undo change

File: webapi/src/main/java/org/sakaiproject/webapi/controllers/CalendarController.java [hunk 4]
```diff
@@ -54,6 +54,9 @@ public class CalendarController extends AbstractSakaiApiController {
     @Autowired
     private EntityManager entityManager;
 
+	@Autowired
```
Comment (jesusmmp) [inline webapi/src/main/java/org/sakaiproject/webapi/controllers/CalendarController.java:57]: Spacing

**Comment by jesusmmp** [comment]
Some little changes. Also it looks good!

**Comment by adrianfish** [comment]
@jesusmmp Thanks for the review :)

I'm having an issue with a test failing, and it's driving me crazy. It does not seem to be caused by any of my changes.

**Comment by ottenhoff** [comment]
> I'm having an issue with a test failing, and it's driving me crazy. It does not seem to be caused by any of my changes.

new API method around that time change with two instants ...

**Comment by adrianfish** [comment]
@ottenhoff It's building on github, tests are passing ok. They still fail on my machine though. It's nothing to do with the two Instants method. I thought so at first, but I eliminated that.

**Review by ottenhoff — APPROVED** [review]

File: calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java [hunk 5]
```diff
@@ -333,23 +314,15 @@ public String eventSubscriptionReference(String context, String calendarId, Stri
...
+	@Override
 	public CalendarEventVector getEvents(List references, TimeRange range, boolean reverseOrder) {
 
 		CalendarEventVector calendarEventVector = null;
 
-		if (references != null && range != null)
-		{
+		if (references != null) {
+			if (range == null) {
+				range = m_timeService.newTimeRange(Instant.EPOCH, Instant.MAX);
```
Comment (ottenhoff) [inline calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java:324]: @ern omg

Comment (adrianfish) [reply calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java:324]: Like, everything. If you don't supply a range, you get everything.

Comment (adrianfish) [reply calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java:324]: Is it causing an issue?

Comment (ottenhoff) [reply calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java:324]: PR incoming ... the problem is that this isn't rows in a database, it's generating them (e.g., weekly meetings)

**Pull request merged**